cli/tools/test/mod.rs

```diff
@@ -289,6 +289,11 @@ impl From<&TestDescription> for TestFailureDescription {
   }
 }
 
+#[derive(Debug, Clone, PartialEq, Deserialize)]
```

Comment:
```suggestion
#[derive(Debug, Clone, PartialEq)]
```

Not needed?

Reply:
Oops, I forgot to delete that setting during trial and error, thanks.